src/pages/MainPage.tsx

```
@@ -1,81 +1,284 @@
+    return new Promise<void>((resolve, reject) => {
+      // 이미 로드된 경우 즉시 완료 처리
+      if (window.naver && window.naver.maps) {
+        resolve();
+        return;
+      }
+      
+      // 로딩 중인 경우 완료될 때까지 대기
+      if (isNaverScriptLoading) {
+        const checkInterval = setInterval(() => {
+          if (window.naver && window.naver.maps) {
+            clearInterval(checkInterval);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)